understanding/20/use-of-color.html

```diff
@@ -44,6 +44,43 @@ <h2>Intent of Use of Color</h2>
          </p>
          
       </div>
+
+
+      <div class="note">
+
+         <p>If content is conveyed through the use of colors that differ not only in their hue,
+            but that also have a significant difference in lightness, then this counts as an additional
+            visual distinction, as long as the difference in contrast between the colors is 3:1 or greater.
+            For example, a light green and a dark red differ <strong>both</strong> by color (hue)
+            <strong>and</strong> by lightness, so they would pass if the contrast difference is at least 3:1.
+            Similarly, if content is distinguished by inverting an element's foreground and background colors,
+            this would pass (again, assuming that the foreground and background colors have a sufficient contrast).
+         </p>
+         <p>However, if content relies on the user's ability to accurately perceive a particular color (either
+            implicitly - e.g. green for success, red for failure - or explicitly - e.g. "the blue items represent
+            the most recently updated entries"), an additional visual indicator will be required, regardless of
+            the relative contrast between those colors.
+         </p>
+
+      </div>
+
+      <div class="note">
+
+         <p>This criterion does not directly address the needs of users with assistive technologies.
+            It aims to ensure that sighted users who cannot perceive color can still understand content.
```

Comment:
Somewhat agree with Andy's comments, "cannot perceive color" is very binary, suggest "users who cannot distinguish between colors".

Reply:
Yes, though even then it's "users who cannot distinguish between **some** colors".   

While normal vision can see millions of colors, most CVD types can still see hundreds of thousands of colors or more.

Reply:
done (both here and in the other places where the understanding document talked about "perceive color")